### DISCUSSION TITLE
W801 http server

### DISCUSSION THREAD

**Discussion by jokokiswan** [openLuat/LuatOS#72] [General]
How to do w801 httpserver just like in esp8266 ? are there any samples? i bought several luatOS supported boards, very powerfull and fast. Thank you!

**Comment by chenxuuu** [comment]
luatos for w801/w806/air101/air103 not support wifi...

**Reply by jokokiswan** [reply]
Thank you for your quick answer, for wifi related is luatOS stick to esp32c3 only? Or any similar cheap board?

**Reply by chenxuuu** [reply]
For now yes

**Reply by jokokiswan** [reply]
ok, thank you for all luatOS team!
